### PR TITLE
#165273494 track comments edit history

### PR DESCRIPTION
**Pull request by mwinel**
### **What does this PR do?**
This PR adds the implementation of the track comments edit history.
- Add updates history
- Add unit tests

### **How to test it**
Try it out on postman with this url
`GET/articles/<article_id>/comments/<comment_id>/update_history/`

Sample Response
```
"update_history": [
     {
         "id": 2,
         "body": "It was not fun at all, edna3",
         "date": "2019-05-09T08:59:42.411673Z",
         "updated_at": "2019-05-09T08:59:42.397637Z"
     },
     {
         "id": 1,
         "body": "It was not fun at all, edna",
         "date": "2019-04-10T08:59:42.411673Z",
         "updated_at": "2019-04-10T08:59:42.397637Z"
     },
]
```
### **Related user stories**
[#165273494](https://www.pivotaltracker.com/story/show/165273494)

### CODE REVIEW

**Review by KabohaJeanMark — APPROVED** [review]
This LGTM. I have tried it out too and it is working fine.

**Review by sanya-kenneth — APPROVED** [review]
Good work Nelson

File: authors/apps/comments/views.py [hunk 1]
```diff
@@ -175,3 +176,40 @@ def post(self, request, **kwargs):
...
+    on a particular article.
+    """
+
+    permission_classes = (IsAuthenticated, )
+
+    def get(self, request, pk, article_id, **kwargs):
+        """
+        Return all comment update history.
+        """
+        get_article(article_id)
```
Comment (archibishop) [inline authors/apps/comments/views.py]: @mwinel Do we need this here.Is it being used anywhere??

File: authors/apps/comments/views.py [hunk 2]
```diff
@@ -175,3 +176,40 @@ def post(self, request, **kwargs):
...
+        comment = get_comment(pk)
+        comment_history = comment.history.all()
```
Comment (archibishop) [inline authors/apps/comments/views.py]: @mwinel We could override the `get_query_set` method and place this there.

File: authors/apps/comments/views.py [hunk 3]
```diff
@@ -175,3 +176,40 @@ def post(self, request, **kwargs):
...
+        update_history = []
+        for update in list(comment_history):
```
Comment (archibishop) [inline authors/apps/comments/views.py]: @mwinel If we can implement the method suggested above there will be no need for this.

File: authors/apps/comments/models.py [hunk 4]
```diff
@@ -1,5 +1,6 @@
 from django.db import models
 
+from simple_history.models import HistoricalRecords
```
Comment (archibishop) [inline authors/apps/comments/models.py:3]: @mwinel Please test out this pr and ensure it allows you to return unlimited number of edits and is not limited to only 4 or 5.

Comment (mwinel) [reply authors/apps/comments/models.py:3]: Yes it does.

File: authors/apps/comments/views.py (hunk 1 above)
Comment (mwinel) [reply authors/apps/comments/views.py]: we are using it to get the article given its id.

**Review by archibishop — APPROVED** [review]
LGTM @mwinel I am merging this.

**Pull request merged**